tests/uaa_configs.py

```diff
@@ -155,5 +147,15 @@
                         'FUBiZx5SOMxvS2nqwwDzK6sqX2Hx2i2\nadgJjtIqgA==',
         'url': 'http://paas.localhost:8080/uaa-security',
         'tags': ['xsuaa_application_plan']
+    },
+    'uaa_no_verification_key': {
+        'clientid': 'sb-xssectest',
+        'xsappname': 'uaa',
+        'identityzone': 'test-idz-name',
+        'identityzoneid': 'test-idz',
+        'clientsecret': 'z431EZmJWiuA/yINKXGewGR/wo99JKiVKAzG7yRyUHld'
+                        'IFUBiZx5SOMxvS2nqwwDzK6sqX2Hx2i2\nadgJjtIqgA==',
+        'url': 'https://lu356076.dhcp.wdf.sap.corp:30332/uaa-security',
```

Comment:
try to omit server names

Reply:
you mean: lu356076.dhcp.wdf.sap.corp? 
Then I should remove them from all test files not only the config I added, right?

Reply:
yes